app/scripts/contentscript.js

```diff
@@ -191,6 +193,59 @@ const insertProgressBar = (activity, forgotten) => {
     document.getElementById("forg").textContent= forgotten;
 }
 
+/*
+ * Creates the progress bar function
+ */
+const ProgressBarFunction = (partial, full, text) => {
+    let progressbar = document.createElement('div')
+    progressbar.innerHTML = progressbarfunction()
+    //Calculate percentage bar
+    let generic_rate = (partial*100)/full;
+
+    //Add div to the page main class
+    document.getElementsByClassName('container new-discussion-timeline experiment-repo-nav')[0]
+        .appendChild(progressbar)
+    document.documentElement.style
+        .setProperty('--progressfunction', generic_rate);
+
+    //Create table to format the description
+    document.getElementById("barfunction").innerHTML = [
+    '<div style="float: justify">',
+    '<TABLE BORDER=0>',
+        '<TR>',
+        '<h1 id="text" style="text-align:center; font-size: 20px"></h1>',
```

Comment:
On [prototype](https://cjjcastro.gitlab.io/2019-1-hubcare-docs/project/prototype/), the title is aligned on left, not on center.